Lightyear/Core.idr

```diff
@@ -153,4 +153,13 @@ satisfy : (Monad m, Stream tok str)
                    => (tok -> Bool)
                    -> ParserT str m tok
 satisfy p = satisfyMaybe (\t => if p t then Just t else Nothing)
+
+||| Succeed if and only if the argument parser fails.
+notFollowedBy : ParserT str m tok -> ParserT str m ()
```

Comment:
It seems strange to me to add something only so as to be able to deprecate it. Perhaps referring to `notFollowedBy` in the docs, where `:apropos` can pick it up, would suffice?